src/compoenents/CustomerMailDetailPannel/IoTHistoryTb.tsx

```diff
@@ -1,77 +1,63 @@
-
 import React, { useEffect, useState } from 'react';
-import '../../assets/styles/ioTHistoryTb.css';
+import ReusableTable from '../CustomerMailDetailPannel/ReusableTable';
 
-interface HistoryData {
-  name: string;
-  organization: string;
-  estate: string;
-  yieldStressDisease: string;
+interface IoTData {
+  id: number;
+  recordedDate: string;
+  temperature: string;
+  humidity: string;
+  uvLevel: string;
+  soilMoisture: string;
+  pressure: string;
+  altitude: string;
 }
 
 const IoTHistoryTb: React.FC = () => {
```

Comment:
In React, it's a common convention to name interfaces for component props with the component's name followed by "Props."
ex: if the component is named IoTHistoryTb, the corresponding interface should be named IoTHistoryTbProps